vcl/haiku/window/salframe.cxx

```diff
@@ -462,6 +463,25 @@ void HaikuSalFrame::ToTop( SalFrameToTop nFlags )
     GetSalData()->mpFirstInstance->PostUserEvent(this, SalEvent::Paint, aPEvt);
     //mpPrivate->mpWindow->Sync();
     //Invalidate(rect);
+    // FIXME which child should be focused
```

Comment:
I don't think it's important - HaikuWindow should have one (and only one) HaikuView which renders everything.